usr/sbin/p11sak/p11sak.c

```diff
@@ -483,6 +489,9 @@ static void print_gen_aes_help(void)
             "      --label LABEL                           key label LABEL to be listed\n");
     printf(
             "      --attr [M R L S E D G V W U A X N]      set key attributes\n");
+    printf( "      -c, --custom-attr [attrs] \n");
+    printf( "             for more info type:\n");
+    printf( "             man p11sak\n");
```

Comment:
....for more info, see the p11sak man page.

Reply:
Changed to see man 1 p11sak